helper/metricsutil/metricsutil.go

```diff
@@ -16,6 +16,8 @@ import (
 const (
 	OpenMetricsMIMEType = "application/openmetrics-text"
 
+	PrometheusSchemaMIMEType = "prometheus/telemetry"
```

Comment:
This was added, but doesn't seem to be used anywhere

Reply:
Good catch! Fixed